EditContext/explainer.md

```diff
@@ -1,19 +1,19 @@
+The EditContext is a new API that simplifies the process of integrating a web app with advanced text input methods (such as VK shape-writing, Handwriting panels, speech recognition, IME Compositions etc), improves accessibility and performance, and unlocks new capabilities for web-based editors.
 
 ## Motivation
-The web platform provides out-of-the-box editing experiences for single lines of plain-text (input), small amounts of multi-line plain-text (textarea) and a starting point for building an HTML document editing experience (contenteditable elements).  
+Today the web platform provides editing experience for single/multi line plain text (input element/ textarea) and support HTML editing through contenteditable elements.
 
-Each of the editable elements provided by the web platform comes with built-in editing behaviors that are often inadequate to power the desired editing experience. As a result, web-based editors don't incorporate the web platform's editable elements into their view. Unfortunately, the only API provided by the web platform today to enable advanced text input experiences is to place an editable element in the DOM and focus it.
+Each of the editable elements provided by the web platform comes with built-in editing behaviors that are often inadequate to power the desired rich editing experience. As a result, web-based editors don't incorporate the web platform's editable elements into their view. Unfortunately, the only API provided by the web platform today to enable advanced text input experiences (like IME compositions, VK shape-writing etc) is to place an editable element in the DOM and focus it.
 
 This contradiction of needing an editable element, but not wanting it to be visible, leads web-based editors to create hidden editable elements to facilitate text input.  This approach negatively impacts accessibility and increases complexity, leading to buggy behavior.
 
 An alternative is to incorporate a contenteditable element into the view of the editor, regardless of whether the editor is editing an HTML document.  This approach limits the editor's flexibilty in modifying the view, since the view is also powering the text input experience.
 
 ## Real-world Examples of Text Input Issues in Top Sites and Frameworks
 ### Accessibility Issues in the Monaco Editor
-[This video](https://www.youtube.com/watch?v=xzC86EG9lPo) demos Windows Narrator reading from a hidden textarea element in the Monaco editor and compares it with the intended experience by showing Narrator reading text from CKEditor, which uses a contenteditable element as part of its view.
+As an example, consider the case of the windows narrator reading from Monaco editor which uses a hidden textarea element versus CKEditor which uses a contenteditable element directly as a part of its view ([Video link](https://www.youtube.com/watch?v=xzC86EG9lPo)).

```

Comment:
I'd keep the existing text here too.  I think its important to say that CKEditor is showing the intended experience.  Also it seems like the original text transitions better to the following paragraph.  Lastly there are some punctuation and grammar errors in the new text.